### DISCUSSION TITLE
What are the factors that affect population simulation?

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#1061] [Questions & Problems]
Hello
I have a question.
![屏幕截图 2021-12-25 191902](https://user-images.githubusercontent.com/81560924/147384318-a8942980-a8ce-4b11-b9d5-a0048626a6e1.jpg)
This is a normal omeprazole metabolism
![屏幕截图 2021-12-25 191923](https://user-images.githubusercontent.com/81560924/147384337-2f03cd29-044d-4f9d-8c8d-b35bb6a7b379.jpg)
This is a simulation of omeprazole with CYP2C19 inhibited.You can see that when CYP2C19 is suppressed, omeprazole concentration is higher than normal.
But the results were reversed in population simulations.
![屏幕截图 2021-12-25 192631](https://user-images.githubusercontent.com/81560924/147384447-eb1db340-4580-40cc-afc3-7284a079763c.jpg)
![屏幕截图 2021-12-25 192647](https://user-images.githubusercontent.com/81560924/147384449-e3e42ce7-1c70-40ae-9b5c-30b17b1108d1.jpg)
Why is this the case?

**Comment by StephanSchaller** [comment]
@Achineseman , all of your simulations show suppression of CYP2C19.
But it looks like their CYP2C19 baselines are different and possibly also the baseline metabolism rate constant. It is hard to evaluate what the differences are if we do not know the exact specifications for each of your simulations (ideally the original project files).

Best,
Stephan
